fedimint-core/src/time.rs

```diff
@@ -12,3 +12,10 @@ pub fn now() -> SystemTime {
     SystemTime::UNIX_EPOCH
         + std::time::Duration::from_secs_f64(js_sys::Date::new_0().get_time() / 1000.)
 }
+
+/// Returns the duration since the Unix epoch
+pub fn duration_since_epoch() -> std::time::Duration {
+    now()
+        .duration_since(SystemTime::UNIX_EPOCH)
+        .expect("time to work")
```

Comment:
nit: I avoided such expects more recently since a horribly misconfigured system might have a time from the past.

Reply:
If your system is so misconfigured you should expect things to blow up

Reply:
There's some room here for maicious input causing panics, I guess.

Reply:
Actually - for this to panic, the `now()` would have to return a date before unix epoch... ? Which is probably impossible at least on Linux, because the representation of time start with unix epoch?